Tasks/VsTestV2/inputparser.ts

```diff
@@ -166,6 +166,31 @@ function getTestReportingSettings(inputDataContract : idc.InputDataContract) : i
     inputDataContract.TestReportingSettings.TestRunTitle = tl.getInput('testRunTitle');
     inputDataContract.TestReportingSettings.TestRunSystem = 'VSTS - vstest';
 
+    const resultsDir = path.resolve(tl.getVariable('Agent.TempDirectory'), tl.getInput('resultsFolder'));
+    inputDataContract.TestReportingSettings.TestResultsDirectory =  resultsDir;
+    tl.debug("TestResultsFolder: " + resultsDir);
+    
+    if (resultsDir.startsWith(tl.getVariable('Agent.TempDirectory') + 'TestResults'))
+    {   
+        ci.publishEvent({ 'TestResultsFolderUi': '$(Agent.TempDirectory)/TestResults' } );
```

Comment:
hey, can you add a customer feature name for this?

Other this event will have the same feature name as all the others and filtering on kusto won't be easy.